docs/development/deploying-locally/index.md

```diff
@@ -1,11 +1,49 @@
 # Deploying locally
 
-REANA can be easily deployed locally on your laptop using _Kubernetes in Docker_ ([kind](https://kind.sigs.k8s.io/)).
-Useful for REANA platform development.
+## For researchers
 
-## Instructions
+If you are a researcher and would like to try out running a small REANA cluster on your laptop, you can proceed as follows.
```

Comment:
```suggestion
If you are a researcher and would like to try out REANA running on a small cluster on your laptop, you can proceed as follows.
```

Reply:
Hmm, this may be perhaps confusing WRT running client-vs-cluster. I'll change to "...try out deploying..." which will hopefully make it clearer.